strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt

```diff
@@ -225,3 +225,11 @@ fun <T : CharSequence> Builder<T>.isNotBlank(): Builder<T> =
  */
 val <T : CharSequence> Builder<T>.length: Builder<Int>
   get() = get(CharSequence::length)
+
+/**
+ * trims a CharSequence
+ *
+ * @see CharSequence.trim
+ */
+fun <T : CharSequence> Builder<T>.trim(): Builder<CharSequence> =
```

Comment:
This should return `Builder<T>`

Reply:
`CharSequence.trim` returns `CharSequence`, but i added a version for string to.

Reply:
ah. Thanks.